### ISSUE TITLE
Readthedocs build is failing.

### ISSUE THREAD

**Issue by abhisrkckl** [nanograv/PINT#1623]
The RTD builds for two of my PRs (independent of each other) keeps failing. The error message I get is the following. It's the same in both cases.

> Running Sphinx v7.2.2
> 
> Traceback (most recent call last):
>   File "/home/docs/checkouts/readthedocs.org/user_builds/nanograv-pint/envs/1621/lib/python3.11/site-packages/sphinx/config.py", line 358, in eval_config_file
>     exec(code, namespace)  # NoQA: S102
>     ^^^^^^^^^^^^^^^^^^^^^
>   File "/home/docs/checkouts/readthedocs.org/user_builds/nanograv-pint/checkouts/1621/docs/conf.py", line 395, in <module>
>     'html_theme': html_theme,
>                   ^^^^^^^^^^
> NameError: name 'html_theme' is not defined
> 
> The above exception was the direct cause of the following exception:
> 
> Traceback (most recent call last):
>   File "/home/docs/checkouts/readthedocs.org/user_builds/nanograv-pint/envs/1621/lib/python3.11/site-packages/sphinx/cmd/build.py", line 293, in build_main
>     app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
>           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/home/docs/checkouts/readthedocs.org/user_builds/nanograv-pint/envs/1621/lib/python3.11/site-packages/sphinx/application.py", line 211, in __init__
>     self.config = Config.read(self.confdir, confoverrides or {}, self.tags)
>                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/home/docs/checkouts/readthedocs.org/user_builds/nanograv-pint/envs/1621/lib/python3.11/site-packages/sphinx/config.py", line 181, in read
>     namespace = eval_config_file(filename, tags)
>                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/home/docs/checkouts/readthedocs.org/user_builds/nanograv-pint/envs/1621/lib/python3.11/site-packages/sphinx/config.py", line 371, in eval_config_file
>     raise ConfigError(msg % traceback.format_exc()) from exc
> sphinx.errors.ConfigError: There is a programmable error in your configuration file:
> 
> Traceback (most recent call last):
>   File "/home/docs/checkouts/readthedocs.org/user_builds/nanograv-pint/envs/1621/lib/python3.11/site-packages/sphinx/config.py", line 358, in eval_config_file
>     exec(code, namespace)  # NoQA: S102
>     ^^^^^^^^^^^^^^^^^^^^^
>   File "/home/docs/checkouts/readthedocs.org/user_builds/nanograv-pint/checkouts/1621/docs/conf.py", line 395, in <module>
>     'html_theme': html_theme,
>                   ^^^^^^^^^^
> NameError: name 'html_theme' is not defined
> 
> 
> Configuration error:
> There is a programmable error in your configuration file:
> 
> Traceback (most recent call last):
>   File "/home/docs/checkouts/readthedocs.org/user_builds/nanograv-pint/envs/1621/lib/python3.11/site-packages/sphinx/config.py", line 358, in eval_config_file
>     exec(code, namespace)  # NoQA: S102
>     ^^^^^^^^^^^^^^^^^^^^^
>   File "/home/docs/checkouts/readthedocs.org/user_builds/nanograv-pint/checkouts/1621/docs/conf.py", line 395, in <module>
>     'html_theme': html_theme,
>                   ^^^^^^^^^^
> NameError: name 'html_theme' is not defined

**Comment by dlakaplan** [comment]
The lines in our `conf.py` file are:
```
if not on_rtd:  # only import and set the theme if we're building docs locally
    html_theme = "sphinx_rtd_theme"
```
so if this is happening on RTD then it shouldn't be setting the theme explicitly.  I'm also confused because the line giving the error doesn't appear on our `conf.py`.

**Comment by dlakaplan** [comment]
I see that sphinx just released 2 versions last week:
https://pypi.org/project/Sphinx/#history
I wonder if we try pinning to an earlier version?

**Comment by dlakaplan** [comment]
Actually I don't think so: builds from last week also used 7.2.2

**Comment by abhisrkckl** [comment]
fixed in #1627
